apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRoutes.kt

```diff
@@ -34,10 +36,11 @@ fun Route.trygdetid(trygdetidService: TrygdetidService, behandlingKlient: Behand
             }
         }
 
-        post {
+        post("{$SAKID_CALL_PARAMETER}") {
```

Comment:
Ref kommentaren lenger opp - kanskje vi kan hente sakType fra behandling i service? Da slipper vi også risikoen for at noen kan sende behandlingId og sakId som ikke hører sammen